addons/addon-base-raas-ui/packages/base-raas-ui/src/helpers/api.js

```diff
@@ -69,6 +69,10 @@ function getAllAccountsPermissionStatus() {
   return httpApiGet(`api/aws-accounts/permissions`);
 }
 
+function getAccountPermissionCfnTemplate(accountUUID) {
```

Comment:
Nit: Can we update the name to getAccountOnboardCfnTemplate? The template does include more resources than just permissions.

Reply:
Yeah, Jeet mentioned something similar and I'm now thinking I should probably change the naming of it across the board...

Reply:
For now I'll just change it here since the other calls are specifically about permissions